src/apiRESTTest/hibernateutil/HibernateUtil.java

```diff
@@ -0,0 +1,29 @@
+package apiRESTTest.hibernateutil;
+
+
+import org.apache.log4j.BasicConfigurator;
+import org.hibernate.SessionFactory;
+import org.hibernate.cfg.Configuration;
+
+public class HibernateUtil {
+
+	private static SessionFactory factory;
+
+	public SessionFactory config() {
+
+		BasicConfigurator.configure();
+
+		try {
+
+			factory = new Configuration().configure().buildSessionFactory();
+
+		} catch (Throwable ex) {
+			System.err.println("Failed to create sessionFactory object." + ex);
```

Comment:
Ideally this exception should be logged, not printed out.

Reply:
I will adress the loggers all the the same time when I start to implement them, if there is no problem.